syntax = "proto3";

package ffmpeg;

service FFmpegService {
  rpc ExecuteCommand (CommandRequest) returns (stream CommandResponse);
}

message CommandRequest {
  string command = 1;
}

message CommandResponse {
  string output = 1;
  int32 exit_code = 2;
  string stream = 3; // Added field to indicate stream type
}